core/src/epicli/cli/engine/ansible/AnsibleRunner.py

```diff
@@ -47,11 +47,20 @@ def run(self):
 
         self.ansible_vars_generator.run()
 
+        repository_setup_play_result = self.ansible_command.run_playbook_with_retries(inventory=inventory_path,
```

Comment:
I think you need to check the offline flag here.  Dont we only want to execute this when we are in online mode?